chapters/en/slides/module1/module1_01.Rmd

```diff
@@ -61,242 +64,122 @@ as we will see later.
-data, we have evolved to instinctively recognize visual patterns and to
-accurately judge properties such as colors and distances between
-objects. Drawing upon this complex build-in apparatus, we can arrive at
-conclusion much faster by glancing at visual representations of data
+Think about it,
+numbers have only been around for about 5,000 years.
+Visual systems on the other hand,
+have undergone refinement for over 500,000,000 years
+by training with a lethal, evolutionary cost function.
+While we need to train ourselves to recognize structure in numerical data,
+we have evolved to instinctively recognize visual patterns
+and to accurately judge properties such as colors and distances between objects.
+Drawing upon this complex build-in apparatus,
+we can arrive at conclusion much faster by glancing at visual representations of data
 rather than numerical representations.
 
 For example, have a look at the
-four sets of numbers in the table below, [put together by statistician
-Francis Anscombe in the
-70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet) Can you see
-how the sets are different from each other by just glancing at the
-table?
+four sets of numbers in the table on the slide.
+This is a slightly modified version of the original,
+[which was put together by statistician Francis Anscombe in the
+70s.](https://en.wikipedia.org/wiki/Anscombe%27s_quartet)
+Can you see how the sets are different from each other
+by just glancing at the table?
 
 ---
 
-## Summary statistics don't necessarily help
-
-### All four sets of numbers are nearly identical in the following statistical properties
-
-<table style="width:80%;">
-<tbody><tr>
-<th>Property
-</th>
-<th>Value
-</th>
-<th>Accuracy
-</th></tr>
-<tr>
-<td><a href="/wiki/Mean" title="Mean">Mean</a> of <i>x</i>
-</td>
-<td>9
-</td>
-<td>exact
-</td></tr>
-<tr>
-<td>Sample <a href="/wiki/Variance" title="Variance">variance</a> of <i>x</i></td>
-<td>11
-</td>
-<td>exact
-</td></tr>
-<tr>
-<td>Mean of <i>y</i>
-</td>
-<td>7.50
-</td>
-<td>to 2 decimal places
-</td></tr>
-<tr>
-<td>Sample variance of <i>y</i></td>
-<td>4.125
-</td>
-<td>±0.003
-</td></tr>
-<tr>
-<td><a href="/wiki/Correlation" class="mw-redirect" title="Correlation">Correlation</a> between <i>x</i> and <i>y</i>
-</td>
-<td>0.816
-</td>
-<td>to 3 decimal places
-</td></tr>
-<tr>
-<td><a href="/wiki/Linear_regression" title="Linear regression">Linear regression</a> line
-</td>
-<td><i>y</i>&#160;=&#160;3.00&#160;+&#160;0.500<i>x</i>
-</td>
-<td>to 2 and 3 decimal places, respectively
-</td></tr>
-<tr>
-<td><a href="/wiki/Coefficient_of_determination" title="Coefficient of determination">Coefficient of determination</a> of the linear regression</td>
-<td>0.67
-</td>
-<td>to 2 decimal places
-</td></tr></tbody></table>
+## Although summary statistics are often useful, they don't tell the whole story
+
+### Set C appears to be slightly different, but are the other sets actually identical?
```

Comment:
We know that the other sets are not actually identical - we just viewed the raw numbers. Can we highlight this here somehow in a useful way? Maybe "The summary statistics, mean and standard deviation, indicate C is very/quite (find the right level of adjective for here) different from the other sets of data in terms of the centre of the sample distribution and the spread of that distribution, and that the remaining 3 sets of data have a similar centre and spread." Or something like that?